pallets/external-validator-slashes/src/lib.rs

```diff
@@ -263,6 +272,44 @@ pub mod pallet {
             NextSlashId::<T>::put(next_slash_id.saturating_add(One::one()));
             Ok(())
         }
+
+        #[pallet::call_index(2)]
+        #[pallet::weight(T::WeightInfo::force_inject_slash())]
+        pub fn root_test_send_msg_to_eth(
```

Comment:
we need to bench this

Reply:
It's only for testing, and ensure_root, what's the point of benchmarking it?

Reply:
I dont think in the future it will be used only for testing, but rather, to inject a forced message if we for somehow skip it